lib/tlIO/CMakeLists.txt

```diff
@@ -67,11 +71,23 @@ if(TLRENDER_FFMPEG)
     list(APPEND SOURCE FFmpeg.cpp FFmpegRead.cpp FFmpegReadAudio.cpp
         FFmpegReadVideo.cpp FFmpegWrite.cpp)
     list(APPEND LIBRARIES_PRIVATE FFmpeg)
+    if(WIN32)
+	set_source_files_properties(FFmpegWrite.cpp
```

Comment:
Why is the C++20 necessary?

Reply:
C++ is needed as MSVC (2019 at least), cannot copy or take an address of these macros:

AVChannelLayout channel_layout = AV_CHANNEL_LAYOUT_STEREO;
return av_channel_layout_copy(dst, &channel_layout);

Linux and macOS accept that already.

Reply:
I think C++20 as a requirement is too early, it's not on the VFX reference platform yet. Hopefully there is another way to do this without requiring C++20.

Reply:
Yes.  I found out that the function av_channel_layout_default(dst, channelCount); handles MONO and STEREO just fine, so that my checks were unneded.  Removed the C++20 dependency.